src/Aspire.Hosting/Extensions/ResourceBuilderExtensions.cs

```diff
@@ -328,6 +328,39 @@ private static void ApplyEndpoints<T>(this IResourceBuilder<T> builder, IResourc
         return builder.WithAnnotation(annotation);
     }
 
+    /// <summary>
+    /// Changes an existing creates a new endpoint if it doesn't exist and invokes callback to modify the defaults.
```

Comment:
The grammar on this comment could use some work.

Reply:
copilot 😄

Reply:
Nope this was all me.